tools/releaseTools.psm1

```diff
@@ -248,4 +248,54 @@ function Get-NewOfficalPackage
     }
 }
 
-Export-ModuleMember -Function Get-ChangeLog, Get-NewOfficalPackage
+##############################
+#.SYNOPSIS
+# Update the version number in code
+#
+#.PARAMETER NewReleaseTag
+# The new Release Tag
+#
+#.PARAMETER NextReleaseTag
+# The next Release Tag
+#
+#.PARAMETER Path
+# The path to the root of where you want to update
+#
+##############################
+function Update-PsVersionInCode
+{
+    param(
+        [Parameter(Mandatory)]
+        [ValidatePattern("^v\d+\.\d+\.\d+(-\w+(\.\d+)?)?$")]
+        [String]
+        $NewReleaseTag,
+
+        [Parameter(Mandatory)]
+        [ValidatePattern("^v\d+\.\d+\.\d+(-\w+(\.\d+)?)?$")]
+        [String]
+        $NextReleaseTag,
+
+        [String]
+        $Path = (Join-path -Path $PSScriptRoot -ChildPath '..')
+    )
+
+    $metaDataPath = (Join-Path -Path $PSScriptRoot -ChildPath 'metadata.json')
+    $metaData = Get-Content -Path $metaDataPath | convertfrom-json
+    $currentTag = $metaData.ReleaseTag
+
+    $currentVersion = $currentTag -replace '^v'
+    $newVersion = $NewReleaseTag -replace '^v'
+    $metaData.NextReleaseTag = $NextReleaseTag
+    Set-Content -path $metaDataPath -Encoding ascii -Force -Value ($metaData | convertto-json)
+
+    Get-ChildItem -Path $Path -Recurse -File |
+        Where-Object {$_.Extension -notin '.icns','.svg' -and $_.NAME -ne 'CHANGELOG.md' -and $_.DirectoryName -notmatch '[\\/]docs|demos[\\/]'} |
```

Comment:
Just to confirm: we are not going to change version strings for docs from now on, right?